uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/contract-entitlements.html

```diff
@@ -0,0 +1,31 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">Contract Entitlements</h3>
+  </div>
+  <div class="table-responsive" ng-show="entitlements.contractsLoaded && entitlements.contracts.length > 0">
+    <table class="table">
+      <thead>
+        <tr>
+          <th style="width:140px">Position</th>
```

Comment:
I think we should move the styles to SCSS. I know I said this before, but I realize that keeping inline styles should be avoided at any cost. Sorry for the confusion.